common/sass/styles/overrides/forms.scss

```diff
@@ -1,86 +1,87 @@
 // Form overrides
-@import "common/sass/variables";
+@import 'common/sass/variables';
 
 label {
-  margin-top: $space-sm;
-  margin-bottom: $space-xs;
-  font-size: $font-size-bump-more;
+	margin-top: $space-sm;
+	margin-bottom: $space-xs;
+	font-size: $font-size-bump-more;
 }
 
 label + .form-control,
 label + input,
 label + textarea {
-  margin-top: 0;
+	margin-top: 0;
 }
 
-input[type="radio"],
-input[type="checkbox"] {
-  margin: 3px 0 0;
-  line-height: normal;
+input[type='radio'],
+input[type='checkbox'] {
+	margin: 3px 0 0;
+	line-height: normal;
 }
 
 input[readonly] {
-  background-color: #fafafa;
-  cursor: text !important;
+	background-color: #fafafa;
+	cursor: text !important;
 }
 
 .form-control {
-  margin-top: $space-sm;
-  margin-bottom: $space-sm;
-  transition: $transition;
+	margin-top: $space-sm;
+	margin-bottom: $space-sm;
+	transition: $transition;
+	z-index: auto !important;
```

Comment:
I'd like to avoid as many `!important`s as possible, what's this about?

Reply:
At some point when I was adding the modal, the z-index of the the `<input type='password'/>` was making it appear on top of the disclaimer. I'm not quite sure why this was occurring but thought it was a bit odd to have a z-index of 2 on all form inputs.

I just took a peek again and your right, its totally unnecessary. The modal has a z-index of 1050 anyway, not quite sure how this slipped past me as I was making the pr. I'll make sure it gets changed back.

Reply:
It was probably when I began writing a modal component before I realized one already existed in the project